tests/net/lldp/src/main.c

```diff
@@ -34,6 +34,7 @@ static struct net_if *iface_posix;
 static struct net_if *iface_fake;
```

Comment:
BTW, the commit message of the LLDP tests should start with "tests: net: lldp: ...."